posthog/temporal/common/logger.py

```diff
@@ -28,6 +28,17 @@ async def bind_temporal_worker_logger(team_id: int, destination: str | None = No
     return logger.new(team_id=team_id, destination=destination, **temporal_context)
 
 
+def bind_temporal_worker_logger_sync(team_id: int, destination: str | None = None) -> FilteringBoundLogger:
+    """Return a bound logger for Temporal Workers."""
+    if not structlog.is_configured():
+        configure_logger()
```

Comment:
We may want to be careful here: This spawns a couple of background tasks that will not run without an async context.